src/package.xml

```diff
@@ -2143,8 +2143,9 @@
         <members>geLabelSectionName</members>
```

Comment:
It looks like "gePS_IntegrationTestInvalidAPIKey" is missing from package.xml

Reply:
not sure how those two components got removed.  fixed

Reply:
fixed